index.js

```diff
@@ -1,93 +1,60 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+import morgan from 'morgan';
+import requestId from 'express-request-id';
```

Comment:
Why do you need this?

Reply:
This is done to prefix a generated request ID to track requests when logging. It is helpful for troubleshooting or debugging